big/int_test.go

```diff
@@ -116,6 +117,36 @@ func TestOperations(t *testing.T) {
 	assert.True(t, ta.Nil())
 }
 
+func TestInt_NilUnmarshal(t *testing.T) {
+
+	parsedStruct := func() (s struct{ Big Int }) {
+		require.NoError(t, json.Unmarshal([]byte("{}"), &s))
+		return s
+	}
```

Comment:
This test is kinda confusing two different things. It would be more clear as two different tests:
- What do you get when you unmarshal nothing into an `Int`. Show exactly that you get a wrapped `nil`, separate from how that behaves.
- How do various operations deal with `nil` parameters. Just use an uninitialised `Int` without jumping through the hoops.

Reply:
@anorth addressed in 0b9c5581fa140aba8fad28550b49584fb59d0b3b